docs/website/pages/index.vue

```diff
@@ -1,40 +1,39 @@
 <template>
   <div id="landing-page" class="c-rich-text">
-    <div class="w-auto md:w-1/2 h-auto mx-auto mt-10 mb-20 px-4">
-      <div class="text-center p-0 m-0">
+    <div class="w-auto lg:w-1/2 h-auto mx-auto md:mt-10 mb-16 px-4">
+      <div class="text-center py-8 m-0">
         <h1>What is Talos?</h1>
       </div>
       <div class="flex flex-wrap justify-center items-center">
-        <div class="px-4 md:px-2 md:w-3/4">
+        <div class="px-4 md:px-2 md:w-3/4 text-xl text-gray-700">
           <p>
             Talos is a modern OS designed to be secure, immutable, and minimal.
             Its purpose is to host Kubernetes clusters, so it is tightly
             integrated with Kubernetes.
           </p>
-          <p>
+          <p class="md:text-base">
```

Comment:
The text above this looks bigger. Is that intentional. Looks a little off to me.

Reply:
Yeah, it was intentional. Does this look too big all around? I thought the regular font was a bit small...

![Screen Shot 2019-10-28 at 11 42 50 AM](https://user-images.githubusercontent.com/423815/67693478-21cd3880-f978-11e9-8cfe-d021e3088566.png)

Reply:
On my screen, the second paragraph looks smaller.

<img width="849" alt="Screen Shot 2019-10-28 at 8 46 56 AM" src="https://user-images.githubusercontent.com/3383143/67693831-99db3480-f95f-11e9-8b77-5714b0e66f37.png">

Reply:
Sorry, yes, my screenshot was a mockup of making both paragraphs the larger size. The PR has the first paragraph bigger. I agree that it looks a bit off. Do you like the bigger font for both paragraphs?

Reply:
I do.